keyboards/kbdfans/kbd19x/info.json

```
@@ -216,6 +216,107 @@
                 {"label":".", "x":17.5, "y":5.5},
                 {"label":"Split Enter (Bottom)", "x":18.5, "y":5.5}
             ]
+        },
+     "LAYOUT_wkl7u": {
```

this needs to match the actual layout name. 
```suggestion
     "LAYOUT_wkl": {
```